frontend/src/testbreed_form.tsx

```diff
@@ -0,0 +1,320 @@
+/**
+ * @file This file contains the form that allows a user to choose mother and father to calculate coefficient of inbreeding
+ */
+import React from 'react'
+import { Link, useRouteMatch, useHistory } from 'react-router-dom'
+
+import { makeStyles } from '@material-ui/core/styles'
+import { Autocomplete } from '@material-ui/lab'
+import { createFilterOptions } from '@material-ui/lab/Autocomplete'
+import { TextField, Button, Typography, Paper } from '@material-ui/core'
+
+import { individualLabel, Individual, LimitedIndividual, Genebank } from '@app/data_context_global'
+import { useDataContext } from '@app/data_context'
+import { useMessageContext } from '@app/message_context'
+import { InbreedingRecommendation } from '@app/testbreed_recommendation'
+
+
+const useStyles = makeStyles({
+  title: {
+    fontSize: '2em',
+    fontWeight: 'bold',
+    paddingLeft: '20px'
+  },
+  buttonBar: {
+    margin: '5px 5px 5px 20px'
+  },
+  form: {
+    width: '100%',
+    height: '100%',
+    padding: '10px',
+    paddingBottom: '90px',
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+  },
+  chooseAncestor: {
+    padding: '2px',
+    margin: '20px',
+    borderSizing: 'border-box',
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+    flexBasis: '45%',
+  },
+  lineBreak: {
+    flexBasis: '100%',
+    height: '0px',
+  },
+  inputAncestor: {
+    padding: '3px',
+    flexBasis: '45%',
+    margin: '6px'
+  },
+  bottomButton: {
+    margin: '15px',
+    padding: '10px',
+    flexBasis: '15%',
+    borderRadius: '5px'
+  },
+})
+
+const inputVariant = 'outlined'
+
+export interface testBreedIndividuals {
+  male: Individual | null,
+  female: Individual | null,
+  maleGF: LimitedIndividual | null,
+  maleGM: LimitedIndividual | null,
+  femaleGF: LimitedIndividual | null,
+  femaleGM: LimitedIndividual | null,
+}
+
+export interface ancestorLabels {
+  maleGFLabel: string,
+  maleGMLabel: string,
+  femaleGFLabel: string,
+  femaleGMLabel: string
+}
+
+const emptyIndividuals: testBreedIndividuals = {
+  male: null,
+  female: null,
+  maleGF: null,
+  maleGM: null,
+  femaleGF: null,
+  femaleGM: null,
+}
+
+const emptyLabels: ancestorLabels = {
+  maleGFLabel: '',
+  maleGMLabel: '',
+  femaleGFLabel: '',
+  femaleGMLabel: ''
+}
+
+/**
```

Comment:
I don't think I understand what this means "added animals unregistered" - that sounds like individuals with no known lineage, which I think means one can't say anything useful (please explain why I'm wrong)?

For the other part, I don't think there should be any problem, anything in the database should be fine I think.

Reply:
I mean animals that not yet have a certificate. They exist but are not in the active population. 
for example, if a herd registers a mating event then this should create x number of individuals but they are not yet "registered" since tfhey do not have a certificate. They are not yet added into the active population but if other herds can find these animals and test-mate with them it can make it much easier to find new breeding pairs. O

We should also extend so animals that were active two years ago also should be findable in this list. 

Or if we can find animals in the big list and click "select for test-mate" like a compare computer thingy you can find in any webshop.